frontend/src/components/readsets/ReadsetsTableColumns.tsx

```diff
@@ -0,0 +1,194 @@
+    return {
+        [ReadsetColumnID.ID]: {
+            columnID: ReadsetColumnID.ID,
+            title: 'ID',
+            dataIndex: ['readset', 'id'],
+            sorter: true,
+            render: (_, { readset }) => {
+                return readset && <div> {readset.id} </div>
+            }
+        },
+        [ReadsetColumnID.SAMPLE_NAME]: {
+            columnID: ReadsetColumnID.SAMPLE_NAME,
+            title: "Sample name",
+            dataIndex: ['readset', 'sample_name'],
+            sorter: true,
+            render: (_, { readset }) => {
+                return readset && <div> {readset.sample_name} </div>
+            }
+        },
+        [ReadsetColumnID.RELEASE_STATUS]: {
+            columnID: ReadsetColumnID.RELEASE_STATUS,
+            title: "Release Status",
+            dataIndex: ['readset', 'release_status'],
+            sorter: true,
+            render: (_, { readset }) => {
+                const { id } = readset;
+                const releaseStatus = releaseStatusOption.specific[id] ?? releaseStatusOption.all ?? readset.release_status
+                const changed = (releaseStatusOption.all && releaseStatusOption.all !== readset.release_status && !releaseStatusOption.specific[id]) || (!releaseStatusOption.all && releaseStatusOption.specific[id])
+                return readset && <Button
+                    disabled={!canReleaseOrBlockFiles}
+                    style={{ color: changed ? "red" : "grey", width: "6em" }}
+                    onClick={() => toggleReleaseStatus(id, OPPOSITE_STATUS[releaseStatus])}>{RELEASE_STATUS_STRING[releaseStatus]}
+                </Button>
+            }
+        },
+        [ReadsetColumnID.LIBRARY_TYPE]: {
+            columnID: ReadsetColumnID.LIBRARY_TYPE,
+            title: "Library Type",
+            dataIndex: ['readset', 'library_type'],
+            sorter: true,
+            render: (_, { readset }) => {
+                return readset && readset.metrics && readset.library_type ? <div> {readset.library_type} </div> : ''
+            }
+        },
+        [ReadsetColumnID.INDEX]: {
+            columnID: ReadsetColumnID.INDEX,
+            title: "Index",
+            dataIndex: ['readset', 'index'],
+            sorter: true,
+            render: (_, { readset }) => {
+                return readset && readset.metrics && readset.index ? <div> {readset.index} </div> : ''
+            }
+        },
+        [ReadsetColumnID.NB_READS]: {
+            columnID: ReadsetColumnID.NB_READS,
+            title: "Number of Reads",
+            dataIndex: ['readset', 'nb_reads'],
+            sorter: true,
+            render: (_, { readset }) => {
+                return readset && readset.metrics && readset.metrics['nb_reads'] ? <div> {Number(readset.metrics['nb_reads'].value_numeric).toFixed(3)} </div> : ''
```

Comment:
The number of reads is a whole number, so it should not have 3 digits after the decimal.